packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3696,7 +3696,7 @@ class InputDecoration {
   ///    rounded rectangle around the input decorator's container.
   final InputBorder? border;
 
-  /// If false [helperText],[errorText], and [counterText] are not displayed,
+  /// If false [helperText],[errorText], and [counterText] are still displayed,
   /// and the opacity of the remaining visual elements is reduced.
```

Comment:
I think the wording of this now suggests that the helperText, errorText, and counterText aren't dimmed, only the rest are. Maybe:
```suggestion
  /// If false the opacity of the visual elements is reduced, including [helperText],[errorText], and [counterText].
```
cc @justinmc for wording opinion.

Reply:
That looks great, thanks @MitchellGoodwin. I tried running this just to double verify that this is what the behavior is, and indeed it's correct.